finesse/hardware/temperature/tc4820.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, name: str, serial: Serial, max_attempts: int = 3) -> None:
 
         self.serial = serial
         self.max_attempts = max_attempts
+        self.max_power = 511
```

Comment:
I'd probably leave this as a constant, but just define the value in this file rather than `config.py`.